api/src/models/PlanDeTransport/modificationBus.js

```diff
@@ -0,0 +1,211 @@
+const mongoose = require("mongoose");
+const mongooseElastic = require("@selego/mongoose-elastic");
+const esClient = require("../../es");
+const patchHistory = require("mongoose-patch-history").default;
+const MODELNAME = "modificationbus";
+
+const Schema = new mongoose.Schema({
+  //Informations de la ligne de bus
+  lineId: {
+    type: String,
+    required: true,
+    documentation: {
+      description: "Id de la ligne de bus",
+    },
+  },
+
+  lineName: {
+    type: String,
+    required: true,
+    documentation: {
+      description: "Nom de la ligne de bus",
+    },
+  },
+
+  // Informations de la demande
+  requestMessage: {
+    type: String,
+    required: true,
+    documentation: {
+      description: "Message de la demande de modification",
+    },
+  },
+
+  requestUserId: {
+    type: String,
+    required: true,
+    documentation: {
+      description: "Id de l'utilisateur ayant fait la demande",
+    },
+  },
+
+  requestUserName: {
+    type: String,
+    required: true,
+    documentation: {
+      description: "Prénom / nom de l'utilisateur ayant fait la demande",
+    },
+  },
+
+  requestUserRole: {
+    type: String,
+    required: true,
+    documentation: {
+      description: "Rôle de l'utilisateur ayant fait la demande",
+    },
+  },
+
+  tags: {
```

Comment:
c'est les tags ou les ids ?
si ce sont des ids je serais pour le mettre 
```suggestion
  tagIds: {
```